apps/core/templates/500.html

```diff
@@ -1,13 +1,14 @@
+{% load i18n %}
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
     <head>
         <meta charset="utf-8" />
-        <title>Internal server error</title>
+        <title>{% trans 'Internal server error' %}</title>
         <meta name="viewport" content="width=device-width, initial-scale=1" />
     </head>
     <body>
-        <h1>Internal server error</h1>
+        <h1>{% trans 'Internal server error' %}</h1>
 
-        <h2>Sorry, there seems to be an error. Please try again soon.</h2>
+        <h2>{% trans 'Sorry, there seems to be an error. Please try again soon.' %}</h2>
     </body>
 </html>
```

Comment:
I'm a bit hesitant to put any 'logic' in 500.html. The server crashed. It might be better to not depend on anything.

I would keep this English.

Reply:
Seems fair to me, but then let’s add a comment to this file to explain why it’s hard-coded to english.